lcpp.lua

```diff
@@ -281,13 +281,11 @@ end
 
 -- returns the number of string occurrences
 local function findn(input, what)
-	local count = 0
 	local offset = 0
 	local _
-	while true do 
+	for count=0,math.huge do

```

Comment:
if it only cosmetic purpose, please do not change this. not only meaningless, but also it will be slightly decrease performance.